apps.toml

```diff
@@ -238,7 +240,7 @@ url = "https://github.com/YunoHost-Apps/bonfire_ynh"
 [borg]
 category = "system_tools"
 level = 7
-potential_alternative_to = [ "Time Machine" ]
+potential_alternative_to = [ "Time Machine", "Veeam" ]
```

Comment:
I would add Restic too

Reply:
I think that listing alternatives that *are* YunoHost apps would create a lot of confusion, also making alternatives listing a nightmare…

Reply:
hmm yes, you're right